webpack/assets/javascripts/react_app/components/common/Loader.js

```diff
@@ -3,32 +3,40 @@ import {STATUS} from '../../constants';
 import MessageBox from './MessageBox';
 import './LoaderStyles.css';
 
-const Loader = ({ status, children, spinnerSize = 'lg' }) => {
-let content;
+const wrapWithLoaderRoot = content => (
+  <div className="loader-root">
+    {content}
+  </div>
+);
+
+export const simpleLoader = (spinnerSize) => wrapWithLoaderRoot(
+  <div className={`spinner spinner-${spinnerSize}`}></div>
```

Comment:
@matanwerbner I submitted a [PR](https://github.com/theforeman/foreman/pull/4343) to create a Spinner component.

It's very simple. If it is merged, you can replace line 13 with
```
<Spinner size={spinnerSize} />
```